File: leetcode/easy/1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map<int,int> m;//ez map sol
        int on = 0;
        for(int i = 0; i < nums.size(); i++){
            on = target - nums[i];
            if(m.count(on)) {return {m[on],i};}
            else {m.insert({nums[i],i});}
        }
        return {};
    }
};

File: leetcode/easy/1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
         for (int i = 0; i < nums.length; i++) { //brute force lol
            int diff = target - nums[i];
            for (int j = i+1; j < nums.length; j++) {
                if (diff == nums[j]) {
                    return new int[]{i, j};
                }
            }
        }
        return new int[] {-1, -1};
        
    }
}